lib/engine/src/export.rs

```diff
@@ -44,17 +46,44 @@ impl From<VMExport> for Export {
     }
 }
 
+/// TODO: rename, etc.
+#[derive(Debug, PartialEq)]
+pub struct ExportFunctionMetadata {
+    /// duplicated here so we can free it....
```

Comment:
This comment needs some more exposition. (Duplicated from where? So who can free it?)

Reply:
Good call, yeah I'll improve the docs.

The field here is never used here outside of `Drop`, it's only there for `Drop`.